ci-operator/jobs/openshift/hypershift/openshift-hypershift-main-periodics.yaml

```diff
@@ -13,6 +13,12 @@ periodics:
     job-release: "4.9"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-hypershift-main-periodics-e2e-aws-periodic
+  reporter_config:
+    slack:
+      channel: '#project-hypershift'
+      job_states_to_report:
+      - failure
+      report_template: Job {{.Spec.Job}} failed.
```

Comment:
Please add something like ` <{{.Status.URL}}|View logs> `, otherwise this will be very frustrating to look into

Reply:
Updated to add logs link and carry on the 🌋 tradition I observe in other jobs